third_party/icu/config.gni

```diff
@@ -5,7 +5,7 @@
 declare_args() {
   # Tells icu to load an external data file rather than rely on the icudata
   # being linked directly into the binary.
-  icu_use_data_file = true
+  # icu_use_data_file = true
```

Comment:
@oxve Is there a way i can make this into a per-platform argument without having to comment it out here ?

Reply:
I think the easiest is adding it into the platforms' `args.gn`. It will require everyone to re-run `gn.py` to apply to their local envs.

Would this be during a transitionary period? It's probably not something we want to keep around in the args.gn files forever.

Maybe @andrewsavage1 has a better idea?

Reply:
Looking at the extent of changes, i think this looks like either all or nothing change. Either we drive it through and make it work on all platforms all at once, or we don't do it.